lib/models/os.ts

```diff
@@ -957,6 +962,41 @@ const getOsModel = function (
 		);
 	};
 
+	/**
+	 * @summary Returns image name for a specific supervisor version for a Device Type
+	 * @name getSupervisorImageForDT
+	 * @public
+	 * @function
+	 * @memberof balena.models.os
+	 *
+	 * @param {Number} deviceTypeId - The Id for the Device Type
+	 * @param {String} version - The semver version string for the supervisor
+	 * @returns {Promise<String>} - Docker image name for the Supervisor version and arch
+	 *
+	 * @example
+	 * const result = balena.models.os.getSupervisorImageForDT(60, 'v12.11.0').then(result => console.log(result))
+	 * // 60 would be raspberrypi4-64 on balena-cloud
+	 *
+	 */
+	const getSupervisorImageForDT = async (
```

Comment:
NIT
```suggestion
	const getSupervisorReleaseByDeviceType = async (
```
which should return the whole record and consumers can grab the part they need.
Keep in mind that this will not include invalidated or draft releases.

Reply:
There's not much more to it, but I get your point, pushing a fix.